solana/program_v2/packages/client/core/src/NetworkService.ts

```diff
@@ -161,6 +161,7 @@ export class NetworkService extends AbstractService {
     authority: PublicKey = this._wallet.publicKey
   ): ServiceBuilder {
     const instructionPromise = this._program.methods
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
       // @ts-ignore
```

Comment:
I guess this is due to bad Anchor IDL stuff. Then it's fine. Please maybe add a quick comment on what is ignored here. e.g. //anchor IDL does not work with nested types (if that is the case here)